components/vf-dropdown/vf-dropdown.njk

```diff
@@ -0,0 +1,25 @@
+{# Make sure any variables are listed inside the following if statement #}
+
+<div class="vf-dropdown" data-vf-js-dropdown style="width:{{width}}">
+  <div class="vf-dropdown__label-container">
+    <div class="vf-dropdown__label" style="text-align: {{labelAlign}}">{{label}}</div>
+    <div>
+      <svg class="arrow-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" height="16" width="16">
```

Comment:
should add an `aria-hidden="true"` to this, as screen readers don't need to read it out..

Reply:
you referring to vf-dropdown__label? I think that needs to be readable by SR?

Reply:
just the svg element. in the interst of time I'll add the commit